### DISCUSSION TITLE
Suspicious Glyph

### DISCUSSION THREAD

**Discussion by object-Object** [HexxyCraft/modpack#10] [General]
Should we disable Suspicious Glyph (the amogus great spell added by HexTweaks that's trivial to brute force and kills you instantly)? Is there any reason _not_ to disable it?

**Comment by Robotgiggle** [comment]
Given that there's already a way to instakill yourself (shame mishap) I don't really see why we need two. Also, it takes up a scroll that could otherwise have something useful on it.

**Comment by LlamaSageR** [comment]
Imagine your first Great Spell being Suspicious Glyph after hours of searching. While Botania's Loomium does alleviate this issue, it still feels bad. However, I recall there being issues with the Hex Casting config not actually disabling spells. A possible solution would be to remove it from loot tables and just have it be crafted. It's already easy to bruteforce, so having it be more easily accessible would not change anything.

It's only a minor issue though. It'd be nice to remove from the loot tables, but not at all necessary.

**Comment by walksanatora** [comment]
what does Boatania Loomium do?
(I though we were adding lootr)
also i *may* work on making it a config that allows downgrading it to a normal spell

**Comment by LlamaSageR** [comment]
Botania's Loomium essentially generates Dungeon loot. A bit more complicated, but that's the main application.

**Comment by walksanatora** [comment]
what i could set `causesBlindDiversion = false` so you cannot use amogus to enlighten

**Comment by LlamaSageR** [comment]
The main issue isn't the triviality of casting it, or the effect it'd have on it's the bloat it adds to the Great Spell pool. There's already ~20 Great Spell scrolls that have to be found, and many of them are not as valued (5 Zeniths, Create Lava, Dispel Rain, etc.), which makes it harder to find the scrolls that are more impactful (Greater Sentinel, GTP, Mind Flay, and so on). 

I stand by what I said before, some toggle to make it not interfere with the Great Spell pool would be ideal.

**Comment by walksanatora** [comment]
okay so config option of `sus_no_enlighten` and `sus_is_normal` will come soon:tm: (until then disable it)
